### ISSUE TITLE
v2.0: Notice "Undefined index" error when adding new content

### ISSUE THREAD

**Issue by xm74** [bludit/bludit#522]
![clipboard04](https://user-images.githubusercontent.com/18723664/31694713-0ef9055a-b3a6-11e7-9668-a1e96f619bc6.jpg)

**Comment by clickwork-git** [comment]
The error means that the counter for the position cannot find the input field.

Is it a fresh installation or an updated installation? Are there any additional plugins installed?

**Comment by xm74** [comment]
@clickwork-git update installation via migrate.php script with few additional plugins (ads4u, addtoany, textarea and yandextools).

**Comment by clickwork-git** [comment]
I will try to reproduce it.

**Comment by anaggh** [comment]
I found the bug. In Bludit v1 Posts do not have a position. The migration script does not set a default position for the post.

@dignajar Should I set a default position for every post in the migration script and what should I set it? 1? 
A temporary workaround is to check if position is set in `dbpages.class.php`
Line 334
```
			if (isset($fields['position']) && $fields['position']>$tmp) {
```

This gets rid of the notices.
